docs/components/connectors/out-of-the-box-connectors/twilio-webhook.md

```diff
@@ -0,0 +1,162 @@
+
+## Make your Twilio Webhook Connector for receiving messages executable
+
+### Fill properties in the **Webhook Configuration** section
+
+1. Choose one of the required methods in the **Webhook method** property. For example, if you know that the webhook will be triggered by the 'POST' method, choose 'POST'. Alternatively, if it is not essential to specify a specific method for the webhook trigger, select 'ANY'.
+2. Configure the **Webhook ID**. By default, the **Webhook ID** is pre-filled with a random value. This value will be part of the Webhook URL. For more details about Twilio Webhook URLs, refer to the section [below on activating the Twilio Webhook Connector by deploying your diagram](#activate-the-twilio-webhook-connector-by-deploying-your-diagram).
+3. Select 'Enabled' in **HMAC authentication** if you want to use HMAC authentication. After that, set the [Twilio Auth Token](https://support.twilio.com/hc/en-us/articles/223136027-Auth-Tokens-and-How-to-Change-Them) as the shared secret key in the **HMAC secret key** field property.
+
+:::note
+It is recommended to use Camunda secrets to store your credentials securely. Refer to the [Camunda secrets documentation](/components/console/manage-clusters/manage-secrets.md) for more details.
+:::
+
+### Fill properties in the **Activation** section
+
+1. (Optional) Configure the **Activation Condition**. For example, if an external message has the body:
+
+   ```
+   {
+     "body": {
+       "ApiVersion": "2010-04-01",
+       "FromCountry": "EU",
+       "Body": "Hello world",
+       "SmsStatus": "received"
+       ...
+     }
+     ...
+   }
+   ```
+
+   the **Activation Condition** value might look like this:
+
+   ```
+   =(request.body.SmsStatus="received")
+   ```
+
+   Leave this field empty to receive all messages every time.
+
+2. When using the **Twilio Webhook Connector** with an **Intermediate Catch Event**, fill in the **Correlation key (process)** and **Correlation key (payload)**.
+
+- **Correlation key (process)** is a FEEL expression that defines the correlation key for the subscription. This corresponds to the **Correlation key** property of a regular **Message Intermediate Catch Event**.
+- **Correlation key (payload)** is a FEEL expression used to extract the correlation key from the incoming message. This expression is evaluated in the Connector Runtime, and the result is used to correlate the message.
+
+For example, if your correlation key is defined with a process variable named `myCorrelationKey`, and you want to correlate by the `FromCountry` property in the request body, which contains:
+
+```
+{
+  "body": {
+    "ApiVersion": "2010-04-01",
+    "FromCountry": "EU",
+    "Body": "Hello world",
+    "SmsStatus": "received"
+    ...
+  }
+  ...
+}
+```
+
+your correlation key settings will look like this:
+
+- **Correlation key (process)**: `=myCorrelationKey`
+- **Correlation key (payload)**: `=request.body.FromCountry`
```

Comment:
is this a valid use case? i would expect something like a message id

Reply:
Message ID is unknown at this point. Probably `Body` (text message) is better fit